addons/website_gengo/static/src/xml/website.gengo.xml

```diff
@@ -149,8 +150,8 @@
                     </b>
                 </main>
                 <footer class="modal-footer">
-                    <button type="button" data-action="set_config" class="btn btn-primary">Submit</button>
                     <button type="button" class="btn btn-secondary" data-action="discard" data-dismiss="modal">Close</button>
+                    <button type="button" data-action="set_config" class="btn btn-primary">Submit</button>
```

Comment:
Are not those UI modals ? With buttons aligned on the left ?

Reply:
No, there are non technical frontend modal
I did it the hard way, not just a search & replace, I tried every fix I made, some modals were not that simple to find how to open, eg portal opportunity address edit \o/

![gengo](https://user-images.githubusercontent.com/30048408/50687611-4ee3df80-1022-11e9-868f-0c4e63a1bfa8.png)

Reply:
As discussed: o_technical_modal on the modal instead, while waiting to use Dialog.js someday

Reply:
Done, I also replaced some website_gengo templates elements. Mostly BS4 'issues':
- H2 were really big (see screenshot previous message) -> h4
- o_technical_modal
- use btn-primary for single button in footer instead of just a link (was weird)